src/components/NotificationCenter/Notification.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Notification() {
```

Comment:
Now, because you pass the prop `notification` to the `Notification` components from the `NotificationCenter` component, you can start using it here to display real life data !